packages/fiddlerclassic.vm/fiddlerclassic.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>fiddlerclassic.vm</id>
-    <version>5.0.20211.20221209</version>
+    <version>5.0.20211.20240417</version>
```

Comment:
This version is wrong, tracked in https://github.com/mandiant/VM-Packages/issues/985 and will be solved in a different PR